generate-release-notes.js

```diff
@@ -10,11 +10,16 @@ async function generateReleaseNotes({
 	exclude = '',
 	commitTemplate = '- {hash} {title}',
 	releaseTemplate = '{commits}\n\n{range}',
-	dateFormat = 'short'
+	dateFormat = 'short',
+	reverseSort = false
 }) {
 	dateFormat = dateFormat.includes('%') ? 'format:' + dateFormat : dateFormat;
 	// Get commits between computed range
 	let {stdout: commits} = await execFile('git', ['log', '--format=%H¬%ad¬%s', '--date=' + dateFormat, range]);
+	const {stdout: commitsReversed} = await execFile('git', ['log', '--format=%H¬%ad¬%s', '--date=' + dateFormat, '--reverse', range]);
```

Comment:
This executes both. It should execute only one. You can just add the `--reverse` flag to the arguments array if necessary

Reply:
I tried adding `--reverse` flag when is `commitsReversed` `true`. If is not, it outputs `''`, that causes error in `git log`...

Reply:
I edited it in https://github.com/fregante/release-with-changelog/pull/43/commits/b3394352b6db666a8f22f1c8e83ce60ad64be1e1.

Reply:
Don't duplicate the whole command, there's only a new flag

Reply:
And how I should it simplify?

Reply:
As I said, I'm quite newbie to JS and Github Actions...

Reply:
I tried
```js
let {stdout: commits} = await execFile('git', ['log', '--format=%H¬%ad¬%s', '--date=' + dateFormat, reverseSort && '--reverse', range])
```
but when is `reverseSort` false, it fails...

Reply:
Still needs to be fixed. It fails because `false` becomes part of that array.

Reply:
Try this or another solution

```js
let {stdout: commits} = await execFile('git', ['log', '--format=%H¬%ad¬%s', '--date=' + dateFormat, reverseSort && '--reverse', range].filter(Boolean))
```

Reply:
Nice, it works 👍